vllm/model_executor/models/baichuan.py

```diff
@@ -366,12 +366,16 @@ def load_weights(self,
                 weight_loader(param, loaded_weight)
 
 
-class BaichuanForCausalLM(BaiChuanBaseForCausalLM):  # baichuan 13b
+class BaichuanForCausalLM(BaiChuanBaseForCausalLM
+                          ):  # baichuan 13b, baichuan2 13b, baichuan2 7b
```

Comment:
Can we make this into two separate lines? This doesn't look good stylistically 😢

Reply:
i think this is how yapf wanted it. but once we switch to black formatting this will be waaaaaaay better

Reply:
you can stamp this PR: https://github.com/vllm-project/vllm/pull/1688 to make it happen!